tools/Scripts/SetupRepository.ps1

```diff
@@ -37,15 +37,23 @@ Set-Location (Split-Path $MyInvocation.MyCommand.Path)
 # for inter-directory linking to build the HolographicCamera.Unity project
 # as well as for the samples to include submodule repository content
 # via directory symlinks.
+Write-Output "Enabling symbolic link directories for the repository."
 git config core.symlinks true
 
-# Ensure that submodules are initialized and cloned
+# Ensure consistent line endings.
+Write-Output "Configuring the repository to use crlf line endings."
+git config core.autocrlf true
+
+# Ensure that submodules are initialized and cloned.
+Write-Output "Updating all submodules."
 git submodule update --init
 
 # If any links were created to the submodules but were broken,
 # restore those symlinks now that the submodules are cloned.
+Write-Output "Updating all symbolic link directories."
 dir "$PSScriptRoot\..\..\" -Recurse -File | ?{$_.LinkType -eq "SymbolicLink" } | Restore-SymbolicLink
 
 # If any links were created before the repository was configured to use
 # symlinks, those links need to be restored.
+Write-Output "Fixing any custom symbolic links."
```

Comment:
>Fixing any custom symbolic links. [](start = 14, length = 33)

I would just lump this second command in with the first as far as reporting status. If you want to describe them separately, I would call the first one "Updating symbolic links into submodules" and the second "Fixing any symbolic links created before symbolic linking was enabled for the repository"